src/app/dashboard/dashboard.component.html

```diff
@@ -1,7 +1,10 @@
-<h1 class="heading">Custom Builders</h1>
+<h1 class="heading">
+  <span>Custom Builders</span>
+  <app-search-bar #sb (onChange) = "onSearch($event)"></app-search-bar>
```

Comment:
can you reomove extra space (onChange)="onSearch($event)"

Reply:
@santoshyadav198613  updated as per your comments